### ISSUE TITLE
Enable selenium 4

### ISSUE THREAD

**Issue by tkoukkari** [serhatbolsu/robotframework-appiumlibrary#334]
Hi,
Latest pypi release has a dependency to selenium version < 4. However in master branch this is no longer required:
`'selenium >= 2.47.1',`

Could this restriction be removed from a pypi release with a new release?

**Comment by axi92** [comment]
This fails for me too!
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
robotframework-appiumlibrary 1.6.3 requires selenium<4,>=2.47.1, but you have selenium 4.0.0 which is incompatible.
```
Installing robotframework-appiumlibrary and robotframework-seleniumlibrary is impossible because of that conflict

**Comment by robinmatz** [comment]
Hi @serhatbolsu ,

could you please release a new version of the library to PyPi?

I noticed that v1.6 already supported selenium >= 4, so there shouldn't be any compatibility issues.

**Comment by serhatbolsu** [comment]
Test fix here please #363
